PaintingTrivia.py

```diff
@@ -161,38 +211,42 @@ def guess_the_saint():
         x = layout_buttons()
         if x:
             return x
+
         try:
-            chosen_painting = int(request.form['chosen_saint'])
+            quiz = pickle.loads(session['quiz'])
+            chosen_saint_id = int(request.form['chosen_saint'])
+            question = quiz.current_question
+            quiz.process_answer(question, chosen_saint_id == question.correct_option.id)
```

Comment:
the quiz should know (and store) what correct answer is. so you shouldn't pass it to the object from the outside